omnibus-arbitrum-mainnet.toml

```diff
@@ -162,114 +224,167 @@ defaultValue = "<%= parseEther('0.000001') %>"
 [setting.price_deviation_tolerance]
 defaultValue = "<%= parseEther('1') %>"
 
-[setting.usdc_staleness_tolerance]
-defaultValue = "<%= String(1 * 3600) %>"
-description = "1 hour"
-
-#
-#
-#
-# USDC
-#
-#
-#
-[setting.usdc_address]
-defaultValue = "0xaf88d065e77c8cC2239327C5EDb3A432268e5831"
-description = "https://developers.circle.com/stablecoins/docs/usdc-on-main-networks"
-
-[setting.pyth_feed_id_usdc]
-defaultValue = "0xeaa020c61cc479712813461ce153894a96a6c00b21ed0cfc2798d1f9a9e9c94a"
-description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
-
-[setting.synth_usdc_max_market_collateral]
+#Collateral Limits
+[setting.max_collateral_limit_arb]
 defaultValue = "<%= parseEther(String(10_000_000)) %>"
-
+[setting.max_collateral_limit_weth]
+defaultValue = "<%= parseEther(String(5_400)) %>"
 [setting.max_collateral_limit_usdc]
 defaultValue = "<%= parseEther(String(10_000_000)) %>"
+[setting.synth_usdc_max_market_collateral]
+defaultValue = "<%= parseEther(String(10_000_000)) %>"
 
-#
-#
-#
-# WETH
-#
-#
-#
+#Token Addresses
+[setting.arb_address]
+defaultValue = "0x912CE59144191C1204E64559FE8253a0e49E6548"
+[setting.USDe_address]
+defaultValue = "0x5d3a1ff2b6bab83b63cd9ad0787074081a52ef34"
+description = "https://coinmarketcap.com/currencies/ethena-usde/"
 [setting.weth_address]
 defaultValue = "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1"
 description = "https://docs.arbitrum.io/build-decentralized-apps/reference/useful-addresses#core-contracts-1"
+[setting.usdc_address]
+defaultValue = "0xaf88d065e77c8cC2239327C5EDb3A432268e5831"
+description = "https://developers.circle.com/stablecoins/docs/usdc-on-main-networks"
+[setting.WSOL_address]
```

Comment:
![image](https://github.com/user-attachments/assets/e2ed587d-9504-4f35-af30-7e54c4798294)

Wrapped wormwhole sol is actually just SOL. Fixing this now.